diff --git a/property_test.prj b/property_test.prj
index d64cb3d..6a9100e 100644
--- a/property_test.prj
+++ b/property_test.prj
@@ -40,8 +40,9 @@ format 75
   mark_for_import
   
-  ncouples 3
+  ncouples 4
   key "butter style" value "invalid"
   key "dummy1" value "property"
   key "dummy2" value "property"
+  key "butter build-dir" value "BUILD"
   comment "Project to test single-directory multiple target use-case for \"butter\" plugout"
   package_ref 128041 // Library
