128041
============================================================
--- a/trunk/test/property_test/128041
+++ b/trunk/test/property_test/128041
@@ -161,4 +161,10 @@ ${namespace_end}"
 	class_ref 128041 // example_lib
       end
+      comment "Library source to test \"butter\" plugout
+
+${butter_generic}
+FLAGS=-DPROGRAM_MESSAGE='\\\"generic1 message\\\"' -DLIBRARY_MESSAGE='\\\"generic2 message\\\"'
+HDR=include/other
+LINK=-s"
     end
 
