Index: proplang.prj
===================================================================
--- proplang.prj	(revision 108)
+++ proplang.prj	(working copy)
@@ -45,7 +45,7 @@
   key "butter ldflags" value "$(OPENMP)"
   key "butter build-dir" value "build"
   key "butter include" value "include/exe"
-  key "butter style" value "unknown"
+  key "butter style" value "boost"
   comment "Project to test single-directory multiple target use-case for \"butter\" plugout
 
 Property Settings:
