Index: proplang.prj
===================================================================
--- proplang.prj	(revision 69)
+++ proplang.prj	(working copy)
@@ -37,7 +37,7 @@
 
   mark_for_import
   
-  ncouples 7
+  ncouples 8
   key "butter base" value ".."
   key "butter version" value "1.4"
   key "butter type" value "shared"
@@ -45,6 +45,7 @@
   key "butter ldflags" value "-Lproj_ldflags"
   key "butter build-dir" value "build"
   key "butter include" value "proj_include include"
+  key "butter style" value "jam"
   comment "Project to test single-directory multiple target use-case for \"butter\" plugout"
   package_ref 128041 // Library
 
